lib/improver/version.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
```

Comment:
Another approach would be to keep the original `VERSION` file in sync by putting some of this logic directly into the shell wrapper. Shelling out to git feels a bit dirty to me,